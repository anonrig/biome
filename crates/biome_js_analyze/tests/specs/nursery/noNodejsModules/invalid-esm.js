import hello from './hello.js'
import fs from 'node:fs'
