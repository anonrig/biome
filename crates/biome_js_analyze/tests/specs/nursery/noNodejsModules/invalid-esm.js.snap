---
source: crates/biome_js_analyze/tests/spec_tests.rs
expression: invalid-esm.js
---
# Input
```js
import hello from './hello.js'
import fs from 'node:fs'

```

# Diagnostics
```
invalid-esm.js:2:16 lint/nursery/noNodejsModules ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ! Using Node.js modules are forbidden.
  
    1 │ import hello from './hello.js'
  > 2 │ import fs from 'node:fs'
      │                ^^^^^^^^^
    3 │ 
  
  i Can be useful for client-side web projects that do not have access to those modules.
  

```


